src/components/Feed.js

```diff
@@ -1,9 +1,234 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import {API_DOMAIN} from './PathList';
+import "bootstrap/dist/css/bootstrap.css";
+import axios from "axios";
+import useToken from "./useToken";
+import like_empty from '../like_empty.png';
+import like_full  from '../like_full.png';
+import dislike_full  from '../dislike_full.png';
+import dislike_empty  from '../dislike_empty.png';
+import comment_empty  from '../comment.png';
+import comment_full  from '../comment_full.png';
 
-const Feed = () => (
-    <div className="container feed">
-    </div>
-);
+
+async function getListOfPosts(AuthStr) {
```

Comment:
Please start var name with lowercase
```suggestion
async function getListOfPosts(authStr) {
```

Reply:
Done